src/App.vue

```diff
@@ -59,6 +59,9 @@ export default class App extends Vue {
         editor: true,
       },
     ],
+      rowClick: function(e: Event, row: Tabulator.RowComponent) {
```

Comment:
I use that App just to fast develop, you don't need update that (Just saying, no problem with that change)